forge-gui/res/cardsfolder/upcoming/guardian_of_ghirapur.txt

```diff
@@ -0,0 +1,11 @@
+Name:Guardian of Ghirapur
+ManaCost:2 W
+Types:Creature Angel
+PT:3/3
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigBlink | TriggerDescription$ When CARDNAME enters the battlefield, exile up to one other target creature or artifact you control. Return it to the battlefield under its owner’s control at the
+SVar:TrigBlink:DB$ ChangeZone | ValidTgts$ Creature.YouCtrl+Other,Artifact.YouCtrl+Other | TgtPrompt$ Select another target creature or artifact you control | Origin$ Battlefield | Destination$ Exile | RememberChanged$ True | SubAbility$ DelTrig | SpellDescription$ Exile another target creature you own. Return it to the battlefield under its owner's control at the beginning of the next end step.
```

Comment:
"up to" not handled

Reply:
Still wrong too

I know you are excited about those new cards but please try to apply the fixes correctly before you resolve